README.md

```diff
@@ -57,56 +56,68 @@ All projects must meet these minimum requirements.
-   [Backdrop License FAQ](https://backdropcms.org/license) for a more detailed
-   explanation.
+1. Any project you create or maintain must include a copy of the GPL-2.0 or
+   GPL-3.0 `LICENSE.txt` file in the root of your repository. The GPL license
+   applies to all code that directly interacts with parts of Backdrop licensed
+   as GPL-2.0 or GPL-3.0. See the [Backdrop License FAQ](https://backdropcms.org/license)
+   for a more detailed explanation.
 
 1. You must confirm that you have the right to distribute any additional code,
    libraries, images, fonts or other assets written or created by any third
-   party with code licensed as GPLv2 or later.
+   party with code licensed as GPL-2.0 or GPL-3.0.
 
 1. Any project you create or maintain must include a `README.md` file containing
    at the least the following:
     1. A description of the project
-    1. Basic documentation
-    1. License information for the project (GPL-2.0)
+    1. Basic documentation or installation instructions.
+    1. License information for the project (GPL-2.0 or GPL-3.0)
     1. License information for any additional assets (SIL OFL fonts, CC-SA
        images, etc)
     1. A list of the current maintainers for the Backdrop project
-    1. Credits acknowledging past maintainers for the Backdrop or Drupal
-       projects
+    1. Credits acknowledging past maintainers, creators, and others worthy.
 
    You may use this [example README.md](https://github.com/backdrop-ops/contrib/blob/master/examples/README.md)
    to get started.
 
-1. You will work with the Backdrop Security Team to address any vulnerabilities
-   in any project you create or maintain, if necessary.
-
 1. Any project you create or maintain will have the GitHub issue tracker enabled
    for official communication.
 
+1. You will work with the `Backdrop Security Team` to address any vulnerabilities
+   in any project you create or maintain, if necessary.
+
+   * If you are not available the day a security release is needed (usually a
+     Wednesday), you understand that the `Backdrop Security Team` may create a
+     security release on your behalf.
+
+1. You will work with the Backdrop Community to review issues in the queue
+   and submitted Pull Requests.
+
+   * If your project has Pull Requests for issues that are tagged "Bug" that have
+     been marked "Reviewed and Tested By the Community" for a period of 2 weeks
+     or more, a member of the `Backdrop Bug Squad` may merge the pull request
+     on your behalf.
+   * If the bug is severe enough the `Backdrop Bug Squad` may also create a new
+     relasse for your project.
+
 1. If any of the above requirements are not met, your access to the Backdrop
    Contrib group -- including all projects and even those that you may have
    originally authored -- may be revoked.
 
-1. If your project becomes abandoned and you do not respond to an issue to grant
+1. If your project becomes Abandoned and you do not respond to an issue to grant
```

Comment:
Instead of repeating the "2 weeks or more" thing, perhaps add a section that has a definition of what makes a project be considered as "Abandoned". Then use that term everywhere else. A "what happens to abandoned projects?" section perhaps? So:

>Abandoned Projects
>------------------
>
>A project is considered abandoned when ...bla bla.
>
>The following actions may be performed by a member of the `Backdrop Bug Squad` without your consent, if your project is deemed abandoned:
>
>- Pull Requests that have been marked "Reviewed and Tested By the Community" which solve bugs may be merged.
>- If a bug is severe enough, a new release for your project may be created.

Reply:
...I see that this was covered further down. Perhaps clean up and remove duplicate mentions of things.

Reply:
these bug squad actions can happen regardless of whether a project is abandoned...